Documentation: Spelling and grammar fixes (#16403)
fleetdm/fleet: docs/Deploy/Deploy-Fleet-on-AWS-ECS.md

#### Fleet migrations

> Migrations in ECS can be achieved (and is recommended) by running [dedicated ECS tasks](https://github.com/fleetdm/fleet/tree/main/infrastructure/dogfood/terraform/aws#migrating-the-db) that runs the `fleet prepare --no-prompt=true db` command. See [terraform for more details](https://github.com/fleetdm/fleet/blob/main/infrastructure/dogfood/terraform/aws/ecs.tf#L261)

Reviewer: Plural "migrations" + "(and is recommended)" still sounds kind of odd... maybe we can cut that part? Just below we say something is not recommended, so I think it's implied that this is the recommended approach.

Also, I think "run" is still correct, because we're talking about multiple tasks running that command:
```suggestion
Migrations in ECS can be achieved by running [dedicated ECS tasks](https://github.com/fleetdm/fleet/tree/main/infrastructure/dogfood/terraform/aws#migrating-the-db) that run the `fleet prepare --no-prompt=true db` command. See [terraform for more details](https://github.com/fleetdm/fleet/blob/main/infrastructure/dogfood/terraform/aws/ecs.tf#L261)
```
